CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ This CHANGELOG follows the format located [here](https://github.com/sensu-plugin
 
 ## [Unreleased]
 
+### Breaking
```

Comment:
For consistency reasons lets use
```suggestion
### Breaking Changes
```
https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#breaking-changes
See my other note (below) and it might make sense to split out the breaking from the non breaking changes here if there are any.